modules/machineset-imds-options.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+:_content-type: CONCEPT
+[id="machineset-imds-options_{context}"]
+= Machine set options for the Amazon EC2 Instance Metadata Service
+
+You can use machine sets to create compute machines that use a specific version of the Amazon EC2 Instance Metadata Service (IMDS). Machine sets can create compute machines that allow the use of both IMDSv1 and link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html[IMDSv2], or compute machines that require the use of IMDSv2.
+
+To change the IMDS configuration for existing compute machines, edit the machine set YAML file that manages those machines. To create new compute machines with your preferred IMDS configuration, create a machine set YAML file with the appropriate values.
+
+The IMDS configuration for control plane machines is set during cluster installation. To change the control plane machine IMDS configuration, you must use the AWS CLI.
+
+[IMPORTANT]
+====
+Before configuring a machine set to create compute machines that require IMDSv2, ensure that your workloads support IMDSv2.
```

Comment:
>your workloads support IMDSv2.

I would be a bit more specific there, maybe:

>your workloads, which interacting with AWS metadata service are support IMDSv2.

Almost all workloads does not care about that IMDSes :)